### PR TITLE
Fix bug in use of pydot.find_graphviz()

### PR DESCRIPTION
**Pull request by sirius27**
pydot.find_graphviz() is deprecated, so bug appear while invoking it. Instead use imp to check if graphviz is imported

### CODE REVIEW

**Comment by cain0095** [comment]
Awesome！

File: keras/utils/visualize_util.py [hunk 1]
```diff
@@ -10,9 +11,11 @@
 except ImportError:
     # Fall back on pydot if necessary.
     import pydot
-if not pydot.find_graphviz():
-    raise ImportError('Failed to import pydot. You must install pydot'
-                      ' and graphviz for `pydotprint` to work.')
+try:
+    imp.find_module('graphviz')
+except:
```
Comment (fchollet) [inline keras/utils/visualize_util.py]: You should except a specific error.

Comment (sirius27) [inline keras/utils/visualize_util.py]: I just followed the original version and used ImportError. What error message do you think is proper.

Comment (fchollet) [inline keras/utils/visualize_util.py]: What exception type does `imp.find_module` throw? Except that.

Comment (sirius27) [inline keras/utils/visualize_util.py]: Fine.I'll start another pull request

Comment (sirius27) [inline keras/utils/visualize_util.py]: Actually imp.find_module throws an ImportError, so I guess there's no need to modify that?

Comment (fchollet) [inline keras/utils/visualize_util.py]: Read this ("When catching exceptions..."): https://www.python.org/dev/peps/pep-0008/#programming-recommendations

**Comment by fchollet** [comment]
Closing outdated PR.

**Pull request closed without merging**